src/github/SyftGithubAction.ts

```diff
@@ -0,0 +1,398 @@
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+import * as github from "@actions/github";
+import * as cache from "@actions/tool-cache";
+import {
+  PullRequestEvent,
+  Release,
+  ReleaseEvent,
+} from "@octokit/webhooks-types";
+import * as fs from "fs";
+import * as os from "os";
+import path from "path";
+import stream from "stream";
+import { SyftOptions } from "../Syft";
```

Comment:
It looks like we're only pulling in `SyftOptions` from `Syft.ts`. The execute is done as part of the `cmd` call. Do we still need the extra Syft interface?

Reply:
nope, was deleted 👍